v2_test.go

```diff
@@ -221,6 +232,18 @@ var validParseAndWriteV2Tests = []struct {
 			rawTLVs:           fixtureTLV,
 		},
 	},
+	{
```

Comment:
Seems we're missing a test with UNSPEC, 0 length and no TLVs. That should address the code coverage decrease coveralls is complaining about.

Reply:
I think I've added (changed) it already. The test with `desc:   "local unspec"`, originally had header LOCAL TCPv4 fixtureIPv4V2, which seems like a copy-paste typo

Reply:
Just added a test for UNSPEC to tlvFormatTests, and the Coveralls is now happy :)